app/views/user_mailer/artist_submission_rejected.html.erb

```diff
@@ -0,0 +1,26 @@
+<%= stylesheet_link_tag 'emails', media: 'all' %>
+
+<table class='padded-email'>
+  <tr>
+    <td>
+      <table class='email-content' align='left'>
+        <tr>
+          <td>
+            <%= render 'shared/email/email_header' %>
+          </td>
+        </tr>
+        <tr>
+          <td class='email-sub-title'>
+            <p>Dear <%= @user.name %>,</p>
+            <p>Thank you for submitting your work to Artsy.</p>
+            <p>Currently we do not work with artists directly. If you are represented by a gallery that would be interested in partnering with Artsy, please encourage the gallery to visit this page.</p>
```

Comment:
Are there supposed to be links to artsy.net here?

Reply:
Added